File: restore_kamish.js
const fs = require('fs');
const path = require('path');

const baseDir = __dirname;
const srcPng = path.join(baseDir, 'kamish-dagger-1-20-1-e1560', 'assets', 'minecraft', 'optifine', 'cit', 'kamish_dagger', 'kamish_dagger.png');
const dstDir = path.join(baseDir, 'src', 'main', 'resources', 'assets', 'corazonmod', 'textures', 'item');

fs.mkdirSync(dstDir, { recursive: true });

fs.copyFileSync(srcPng, path.join(dstDir, 'mafia_dagger.png'));
fs.copyFileSync(srcPng, path.join(dstDir, 'kamish_dagger.png'));

console.log('[RESTORED] Original Kamish Dagger PNG texture restored successfully!');

File: restore_kamish.py
import shutil
import os

base_dir = r"d:\codingan\A-project-corazon"
src_png = os.path.join(base_dir, "kamish-dagger-1-20-1-e1560", "assets", "minecraft", "optifine", "cit", "kamish_dagger", "kamish_dagger.png")
dst_dir = os.path.join(base_dir, "src", "main", "resources", "assets", "corazonmod", "textures", "item")

dst_mafia = os.path.join(dst_dir, "mafia_dagger.png")
dst_kamish = os.path.join(dst_dir, "kamish_dagger.png")

shutil.copyfile(src_png, dst_mafia)
shutil.copyfile(src_png, dst_kamish)

print("[RESTORED] Original Kamish Dagger PNG texture restored successfully!")
